src/OpenSage.Game/Logic/Object/Update/AIUpdate/HackInternetAIUpdate.cs

```diff
@@ -107,7 +106,11 @@ public HackInternetAIUpdateStateMachine(GameObject gameObject, GameEngine gameEn
     internal LogicFrameSpan GetVariableFrames(LogicFrameSpan time, GameEngine gameEngine)
     {
         // take a random float, *2 for 0 - 2, -1 for -1 - 1, *variance for our actual variance factor
```

Comment:
this comment probably isn't needed anymore now that you made the code readable lol